src/view/pages/statement/components/settings/components/membership/MembersSettings.tsx

```diff
@@ -50,6 +55,21 @@ const MembersSettings: FC<StatementSettingsProps> = ({ statement }) => {
 		navigator.share(shareData);
 	}
 
+	const fetchAwaitingUsers = async (): Promise<number> => {
+		const usersCollection = collection(DB, Collections.awaitingUsers);
+		const usersSnapshot = await getDocs(usersCollection);
+		return usersSnapshot.docs.length
+	}
+
+	useEffect(() => {
+		const fetchData = async () => {
+			const count = await fetchAwaitingUsers();
+			setUserCount(count);
+		};

```

Comment:
This function is not necessary. 
You can call fetchAwaitingUser directly and move setUserCount inside that function.